turbo/jsonrpc/eth_receipts.go

```diff
@@ -154,6 +154,11 @@ func (api *APIImpl) GetLogs(ctx context.Context, crit filters.FilterCriteria) (t
 		end = latest
 	}
 
+	endBlockMaxrange := begin + api.LogsMaxRange
+	if end > endBlockMaxrange {
+		return nil, fmt.Errorf("limit exceeded")
```

Comment:
could we change this to "block range too large" or something so it doesn't get confused with a rate limit?